modules/self-managed-node-group/main.tf

```diff
@@ -158,7 +158,7 @@ resource "aws_launch_template" "this" {
       dynamic "spot_options" {
         for_each = lookup(instance_market_options.value, "spot_options", null) != null ? [instance_market_options.value.spot_options] : []
         content {
-          block_duration_minutes         = spot_options.value.block_duration_minutes
+          block_duration_minutes         = lookup(spot_options.value, block_duration_minutes, null)
```

Comment:
block_duration_minutes should be quoted right? 

```
block_duration_minutes = lookup(spot_options.value, "block_duration_minutes", null)
```

Reply:
Ok, opened a pr: https://github.com/terraform-aws-modules/terraform-aws-eks/pull/1881